neqo-transport/src/stats.rs

```diff
@@ -17,6 +21,22 @@ pub struct Stats {
     pub dups_rx: usize,
     /// Dropped datagrams, or parts thereof
     pub dropped_rx: usize,
-    /// resumsion used
+    /// resumption used
     pub resumed: bool,
 }
+
+impl Stats {
+    pub fn init(&mut self, conn_info: String) {
+        self.conn_display_info = conn_info;
+    }
+
+    pub fn pkt_dropped(&mut self, reason: impl AsRef<str>) {
+        self.dropped_rx += 1;
+        qwarn!(
```

Comment:
As dropping packets is likely more common than this, I would drop this to `qinfo!()`.